Automattic/woocommerce-payments
# feat: disable button until payment methods are selected

Fixes #

#### Changes proposed in this Pull Request









Disable payment methods button in modal until at least one payment method has been selected.

For background info: p1622132280144800-slack-CU6SYV31A

#### Testing instructions





- Ensure you have the latest version of the dev tools: `cd docker/wordpress/wp-content/plugins/woocommerce-payments-dev-tools && git checkout trunk && git pull`
- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Ensure you have the "Add UPE additional payment methods" flag enabled in WCPay Dev Tools
- Go to your WCPay Home: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- Click on "add payment method" to display the modal
- The CTA on the modal stays disabled until at least one payment method is checked

-------------------
